procedures/unit-testing-basics.ipf

```diff
@@ -2245,6 +2286,33 @@ static Function AfterTestCase(name)
 
 	DFREF dfr = GetPackageFolder()
 	NVAR/SDFR=dfr assert_count
+	NVAR/SDFR=dfr waveTrackingMode
+
+#if IgorVersion() >= 9.0
+	if(NVAR_Exists(waveTrackingMode))
+		if((waveTrackingMode & UTF_WAVE_TRACKING_LOCAL) == UTF_WAVE_TRACKING_LOCAL)
+			WaveTracking/LOCL count
+			if(V_Flag == 1)
```

Comment:
Introduce a static Constant for the WaveTracking mode check.
What happens if the mode here is not 1? (i.e. the use code hampered with the wavetracking)
Currently you ignore that silently.
I think this should give an error as well, as the test was run with the purpose to check that and if the UTF can't do that because the test case is incompatible it should fail as well.

- Silently ignoring is not an option